### ISSUE TITLE
Superuser status is required for some basic Alyx functions?

### ISSUE THREAD

**Issue by claire-rbblab** [cortex-lab/alyx#657]
I went into MISC>Lab members>my record and gave myself "Superuser status" (Designates that this user has all permissions without explicitly assigning them). With this setting at least one of my previously reported issues was resolved: #649

When the Superuser status box is unchecked and yet all permissions are selected, I have the issues referenced above. So perhaps there are permissions not available in this list that should be added or there is an error in several of the permissions?
![permissions](https://user-images.githubusercontent.com/63722381/91621903-e10cca00-e962-11ea-9b9f-3ca0e99821b8.PNG)

Wondering what the Superuser status is meant for, and whether it is okay to assign all lab members to have this status. What are the permissions that would be essential to restrict in order to prevent accidental deletion of records/data?

**Comment by rossant** [comment]
The permission system currently uses the superuser checkbox to enable some actions that are reserved to the developers and lab managers who are in charge of the mouse colony management. Is your issue that you may have non lab-manager users who should still be able to add/edit animals, litters, etc? I suppose the permission system should be improved by using proper groups rather than the superuser checkbox.

**Comment by claire-rbblab** [comment]
That is correct, we don't currently have a lab manager so it is important that each lab member can add/edit animals, etc. It seems that even if I am not a superuser but I add all of the permissions, there are still certain items that I should be able to do that are not available. Perhaps these are bugs? 

I should have are the abilities to edit litters and breeding pairs based on the permissions I have selected (non-superuser), but when I open a litter or breeding pair record, none of the fields can be updated. I do not receive a specific error message, however.
